modules/cas_cache/utils/utils_rpool.c

```diff
@@ -177,7 +180,10 @@ struct cas_reserve_pool *cas_rpool_create(uint32_t limit, char *name,
 	info.rpool_new = rpool_new;
 	info.allocator_ctx = allocator_ctx;
 
-	for (i = 0; i < cpu_no; i++) {
+	for (i = 0; i < max_cpus; i++) {
+		if (!cpu_online(i))
```

Comment:
I think this case is covered by the fact that I'm kzalloc * max_cpus. If anything we are allocating more memory than possibly needed but it should cover this case. 

An argument can be made that we may allocate memory unnecessarily in most situations but since this is happening only on initialization it would probably require a more complicated mechanism to handle that case. 

What would you recommend?